site/docs/benchmarks.md

```diff
@@ -128,4 +128,8 @@ column used in the filter predicate. The performance is compared to the built-in
 
 ### SparkParquetWritersNestedDataBenchmark
 * A benchmark that evaluates the performance of writing nested Parquet data using Iceberg and Spark Parquet writers. To run this benchmark for either spark-2 or spark-3:
-  `./gradlew :iceberg-spark:iceberg-spark[2|3]:jmh -PjmhIncludeRegex=SparkParquetWritersNestedDataBenchmark -PjmhOutputPath=benchmark/spark-parquet-writers-nested-data-benchmark-result.txt`
+`./gradlew :iceberg-spark:iceberg-spark[2|3]:jmh -PjmhIncludeRegex=SparkParquetWritersNestedDataBenchmark -PjmhOutputPath=benchmark/spark-parquet-writers-nested-data-benchmark-result.txt`
+
+### VectorizedReadParquetDecimalBenchmark
+* A benchmark to compare performance of reading Parquet decimal data using vectorized Iceberg read path and the built-in file source in Spark.
+`./gradlew :iceberg-spark[2|3]:jmh -PjmhIncludeRegex=VectorizedReadParquetDecimalBenchmark -PjmhOutputPath=benchmark/vectorized-read-parquet-decimal-result.txt`
```

Comment:
These benchmarking commands are relatively out of date btw. I’ll open a PR to update them in the files and then you can copy from there if you’d like?

Reply:
Opened a PR with the updated commands for Spark 3.2: https://github.com/apache/iceberg/pull/4328

Reply:
Updated